api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -0,0 +1,141 @@
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+type TestDataItem struct {
+	fieldName     string
+	fieldValue    string
+	fieldMinValue string
+	errorMessage  string
+}
+
+var _ = Describe("SelfNodeRemediationConfig Validation", func() {
+
+	Describe("validate update time fields of SelfNodeRemediationConfig CR", func() {
+		shortTimeTestItems := []TestDataItem{
+			{"PeerApiServerTimeout", "1.2ms", MinDurPeerApiServerTimeout, ErrPeerApiServerTimeout},
```

Comment:
Nit: I suggest using more diverse values instead of 1.2 for all the tests

Reply:
And maybe use a const variable instead of ` 1.2ms`.